### ISSUE TITLE
Bug: Unable to download file

### ISSUE THREAD

**Issue by charmoniumQ** [IQSS/dataverse.harvard.edu#220]
**What steps does it take to reproduce the issue?**

* Which page(s) does it occurs on?

<https://dataverse.harvard.edu/file.xhtml?persistentId=doi:10.7910/DVN/5UMCKK/2FO3JJ&version=2.2>

* What happens?

When I click to download the file, nothing gets downloaded.

Curling the URL directly shows: `{"status":"ERROR","code":404,"message":"Datafile 3080259: Failed to locate and/or open physical file."}`

* To whom does it occur (all users, curators, superusers)?

All users

* What did you expect to happen?

Dataverse would initiate file downloading. At the very list, the browser should show an error message.

**Which version of Dataverse are you using?**

v. 5.13 build 1244-79d6e57

**Comment by landreev** [comment]
Hello, 
This is a genuinely missing file (the record of it exists in the database, but the physical file is missing in storage). This one has been on the radar, or was at some point at least, I found some traces of conversations about it. We believe that the file was never there, it was never saved, never ended up on any backups. Possibly a download that got caught in a server crash. Or a result of some old bug. At some point an attempt was made to contact the author, to see if the file could be re-uploaded, or purged from the database. 
The person who tried communicating with them is no longer part of the project. I don't know what happened, but I'm guessing there was no response. 

So, it's a messy legacy case. There are a few more like this, unfortunately and we have limited resources for dealing with things of this nature. But thanks for reminding us, I will make another effort to see if it can be reuploaded (or even deleted quietly?)

**Comment by sbarbosadataverse** [comment]
@landreev Do we have the record of the others? I will make a way to prioritize finding the data owners.

**Comment by landreev** [comment]
Yeah, I'll need to look into this again and compile an accurate list of such legacy cases with missing physical files. Not something I can produce instantly. 
Danny was still around the last time we tried to address this. I know he was contacting some authors. But not quite sure how many responded etc.

**Comment by stevenwinship** [comment]
See PR for new API to get the missing file info: https://github.com/IQSS/dataverse/pull/11016
